composer.json

```diff
@@ -29,44 +23,44 @@
         "doctrine/annotations": "^1.13",
         "eluceo/ical": "^0.15.0",
         "ezyang/htmlpurifier": "4.13.*",
-        "fideloper/proxy": "^4.4",
         "firebase/php-jwt": "^6.8",
         "fruitcake/laravel-cors": "^2.0",
```

Comment:
@romanetar please review this
Package fruitcake/laravel-cors is abandoned, you should avoid using it. No replacement was suggested.